frameworks/Pascal/mormot/src/raw.pas

```diff
@@ -193,6 +193,7 @@ constructor TRawAsyncServer.Create(
      hsoIncludeDateHeader  // required by TPW General Test Requirements #5
     ] + flags);
   fHttpServer.HttpQueueLength := 10000; // needed e.g. from wrk/ab benchmarks
+  fHttpServer.ServerName := 'M'; // better plaintext performance
```

Comment:
This optimization has been disallowed for other frameworks by the past:

> We expect the Server header to be whatever is normal for the platform or framework. If the framework does not normally provide a Server response header, we nevertheless require that you provide one as this roughly normalizes network load across all implementations.

Reply:
Default Server: header produced by mORMot2 framework is `mORMot2 (Linux)` what is too long. So we override it to short one-letter version, as most other top-rated frameworks did, for example - .net use ["K" instead of "Kestrel"](https://github.com/TechEmpower/FrameworkBenchmarks/blob/master/frameworks/CSharp/aspnetcore/PlatformBenchmarks/BenchmarkApplication.Fortunes.cs#L24)

Reply:
Must have been confused with the `/p` in plaintext routes. Thanks for confirming.

Reply:
It's disallowed to all play with the same rules.
But still not formalized in the rules and verification tests. :(

Reply:
@joanhey - so what your suggestion for `Server` header value? 
To be the same as `webserver` in `benchmark_config.json`, or, if it is equal to "None" as for many frameworks - to be  the same as  `framework` ?

Reply:
The problem is not the name, but the minimum number of characters.
4 chars more in the server header, seems not to much. But if we multiply these 4 extra chars for 6M req/s, exist a big difference.

Reply:
Here is examples of top rated frameworks what use one-letter for `Server:`
 -  may [Server: M](https://github.com/Xudong-Huang/may_minihttp/blob/master/src/response.rs#L108)
 -  .net [Server: K](https://github.com/TechEmpower/FrameworkBenchmarks/blob/master/frameworks/CSharp/aspnetcore/PlatformBenchmarks/BenchmarkApplication.Fortunes.cs#L24)
 - ntex [Server: N](https://github.com/TechEmpower/FrameworkBenchmarks/blob/master/frameworks/Rust/ntex/src/utils.rs#L7)
 and so on....

I don't think all of them can quickly change their implementation. 
For *fair play* we also use `Server: M`  for mORMot

Reply:
Yes, I know.
It's for that, than we need to clarify and add rule and verification.

Reply:
For example, with Nginx you can't change the server name.

Reply:
`Server` header value is changed to `mmt` by [658165b824] .
@joanhey - hope rule will be changed - I propose to use at last 3 character. For example google respond with `server: gws`, so 4 characters is too much. 
And, for sure, we should add validation for new rule, to ensure all participants are in the same situation

Reply:
> 

We need to clarify this, today socketify.py uses `Server: uWebSockets_20`, we can remove Server header or add a minimal size like 3 characters, in my case I can change to `Server: uWS`